src/pages/workspace/accounting/netsuite/export/NetSuiteExportConfigurationPage.tsx

```diff
@@ -33,17 +36,25 @@ import ToggleSettingOptionRow from '@pages/workspace/workflows/ToggleSettingsOpt
 import * as Policy from '@userActions/Policy/Policy';
 import CONST from '@src/CONST';
 import ROUTES from '@src/ROUTES';
+import type SCREENS from '@src/SCREENS';
 
 type MenuItemWithSubscribedSettings = Pick<MenuItem, 'type' | 'description' | 'title' | 'onPress' | 'shouldHide'> & {subscribedSettings?: string[]};
 
 function NetSuiteExportConfigurationPage({policy}: WithPolicyConnectionsProps) {
     const {translate} = useLocalize();
     const styles = useThemeStyles();
-    const policyID = policy?.id ?? '-1';
+    const route = useRoute<PlatformStackRouteProp<SettingsNavigatorParamList, typeof SCREENS.WORKSPACE.ACCOUNTING.NETSUITE_EXPORT>>();
+    const backTo = route?.params?.backTo;
+    const policyID = policy?.id;
     const policyOwner = policy?.owner ?? '';
     const {canUseNetSuiteUSATax} = usePermissions();
 
     const config = policy?.connections?.netsuite?.options.config;
+    const isConnectionShouldBeRemovedFromCompanyCard =
```

Comment:
Maybe lets name it more specifically to what it will cause/ do. It would be great to also add some comment explaining why we need it
```suggestion
    const shouldgoBackToSpecificRoute =
```

Reply:
I discussed with @narefyev91 these changes, and he is out until Friday. These are NAB changes but we should make them for better DX. We can add it to some future fixes to the flow and not hold in this